docker/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM rocker/geospatial:4.1.2
+
+# Disable the annoying bell on WSL2
+RUN sed -i 's/^# set bell-style none$/set bell-style none/' /etc/inputrc
+RUN echo 'set visualbell' >> /root/.vimrc
+
+# Add DOI CA to local CAs so that SSL can work over VPN
+COPY DOIRootCA2.crt /usr/local/share/ca-certificates
+RUN update-ca-certificates
```

Comment:
I would suggest setting the `CURL_CA_BUNDLE` environment variable as well, as [here](https://code.usgs.gov/jross/data-releases-container/-/blob/main/Dockerfile#L10)